config/environment.py

```diff
@@ -57,6 +59,7 @@
     OAUTH_TOKEN = os.environ.get('OAUTH_TOKEN')
     SECRET_KEY = os.environ.get('SECRET_KEY') or 'NO SECRETS'
     HOOK_SECRET_KEY = os.environ.get('HOOK_SECRET_KEY') or 'SECRETS'
+    ANON_REPORTING_ENABLED = False
```

Comment:
Maybe in the future we want to allow localhost anonymous reporting, but for now just set everything to False.